fop/fom.c

```diff
@@ -1258,6 +1286,7 @@ M0_INTERNAL void m0_fom_domain_fini(struct m0_fom_domain *dom)
 	int i;
 
 	m0_locality_chore_fini(&dom->fd_hung_foms_chore);
+	m0_locality_chore_fini(&dom->fd_locm_warn_chore);
```

Comment:
[* style *] Please make finalisation order the same as initialization.

Reply:
Agree, done.